Prevent crash on Android.
--- a/src/widgets/kernel/qtooltip.cpp
+++ b/src/widgets/kernel/qtooltip.cpp
@@ -406,7 +406,9 @@
                                                              QGuiApplication::primaryScreen());
     // a QScreen's handle *should* never be null, so this is a bit paranoid
     if (const QPlatformScreen *platformScreen = screen ? screen->handle() : nullptr) {
-        const QSize cursorSize = QHighDpi::fromNativePixels(platformScreen->cursor()->size(),
+        const auto *cursor = platformScreen->cursor();
+        const auto native_size = cursor ? cursor->size() : QSize(16, 16);
+        const QSize cursorSize = QHighDpi::fromNativePixels(native_size,
                                                             platformScreen);
         QPoint offset(2, cursorSize.height());
         // assuming an arrow shape, we can just move to the side for very large cursors
